AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/consts/physics.py

```diff
@@ -0,0 +1,28 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Holds constants used across both hydra and non-hydra scripts.
+"""
+
+"""
+PhysX Components
+"""
+PHYSX_SHAPE_COLLIDER = "PhysX Shape Collider"
+
+"""
+Shape Components
+"""
+BOX_SHAPE_COMPONENT = "Box Shape"
+CAPSULE_SHAPE_COMPONENT = "Capsule Shape"
+CYLINDER_SHAPE_COMPONENT = "Cylinder Shape"
+POLYGON_PRISM_SHAPE_COMPONENT = "Polygon Prism Shape"
+QUAD_SHAPE_COMPONENT = "Quad Shape"
+SPHERE_SHAPE_COMPONENT = "Sphere Shape"
+
+
+
+
+
```

Comment:
nit: Can we lose these trailing lines to just one at the EOF

Reply:
Removed